### PR TITLE
Issue#205

### PR DESCRIPTION
**Pull request by ErmiasG**
[CU-1zx724j fixes select2 search](https://app.clickup.com/t/1zx724j)

### CODE REVIEW

**Comment by giardiv** [comment]
Task linked: [CU-1zx724j Reset user password doesn't load the list of users](https://app.clickup.com/t/1zx724j)

**Comment by giardiv** [comment]
All good compiling with the front pr as well,
minus the issue you mentioned today regarding the list of users that does not load

**Comment by giardiv** [comment]
All good compiling with the front pr as well,
minus the issue regarding the list of users that does not load

**Comment by ErmiasG** [comment]
This is the fix for the input losing focus. [This](https://github.com/logicalclocks/hopsworks-front/pull/393) fixes users not loading.
reset-pwd

File: src/components/select2/index.tsx [hunk 1]
```diff
@@ -243,7 +223,11 @@ const Select: FC<SelectProps> = ({
         needSwap={needSwap}
       >
         {isOpen && (
-          <DropdownWrapper refEl={containerRef?.current || undefined}>
+          <DropdownWrapper
+            refEl={containerRef?.current || undefined}
```
Comment (esonhey) [inline src/components/select2/index.tsx:227]: nit: no need to include `undefined` since you have already add optional chaining

**Review by esonhey — APPROVED** [review]
Looks good

**Pull request merged**